### PR TITLE
cmd/setup-etcd-environment: fix unmarshalling EtcdScaling error

### PR DESCRIPTION
**Pull request by alaypatel07**
The discovery container was erroring out because of the change in CEO
EtcdScaling data structure. This PR uses the CEO package to import the
EtcdScaling data structure.

Depends on hexfusion/cluster-etcd-operator#42

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -172,3 +172,8 @@ required = [
 [[constraint]]
   name = "github.com/InVisionApp/go-health"
   version = "2.1.0"
+
+[[constraint]]
+  name = "github.com/openshift/cluster-etcd-operator"
+  source = "github.com/alaypatel07/cluster-etcd-operator"
```
Comment (hexfusion) [inline Gopkg.toml]: please, move back to upstream

**Review by hexfusion — COMMENTED** [review]
one small nit

**Comment by alaypatel07** [comment]
@hexfusion done, PTAL

**Review by hexfusion — APPROVED** [review]
/lgtm

**Pull request merged**